src/view/com/post-thread/PostThread.tsx

```diff
@@ -1,9 +1,12 @@
-import React, {useEffect, useRef} from 'react'
-import {useWindowDimensions, View} from 'react-native'
+import React, {useRef} from 'react'
+import {StyleSheet, useWindowDimensions, View} from 'react-native'
 import {runOnJS} from 'react-native-reanimated'
+import Animated from 'react-native-reanimated'
+import {useSafeAreaInsets} from 'react-native-safe-area-context'
 import {AppBskyFeedDefs} from '@atproto/api'
 import {msg, Trans} from '@lingui/macro'
 import {useLingui} from '@lingui/react'
+import {clamp} from 'lodash'
```

Comment:
lodash is a transient dependency from rn-fetch-blob that we should avoid - it should be `import clamp from 'lodash.clamp'`

Reply:
Oh, just released it's not new, it's just moved from another file. We should see about removing that transient dependency though, I have a PR here: https://github.com/bluesky-social/social-app/pull/4553

Reply:
yea but good callout